controllers/controllers.api.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs/promises");
+
+exports.getEndpoints = (req, res, next) => {

```

Comment:
Possibly overkill - but preferably a model is designated to interact with data. However, in this case it is just dealing with a static file so this is fine by me!

Reply:
Thanks for bringing that up and I'll keep it in mind. I was under the assumption models interfaced with the database, not any data that we're storing. I won't bother changing it if it's fine with you though